rp2040-hal/src/usb.rs

```diff
@@ -50,6 +50,85 @@ use usb_device::{
     Result as UsbResult, UsbDirection, UsbError,
 };
 
+unsafe fn errata5_workaround_finish((prev_pads, prev_io_ctrls, start_ts): (u32, u32, u32)) -> bool {
+    let pac = crate::pac::Peripherals::steal();
+    let now = pac.TIMER.timerawl.read().bits();
+    if now.wrapping_sub(start_ts) < 1_000 {
+        return false;
+    }
+    let pads = &pac.PADS_BANK0.gpio[15];
+    let io = &pac.IO_BANK0.gpio[15];
+
+    // Should think we are connected now, tight loop just in case
+    while pac
+        .USBCTRL_REGS
+        .sie_status
+        .read()
+        .connected()
+        .bit_is_clear()
+    {
+        cortex_m::asm::nop();
+    }
+
+    // Switch back to USB phy
+    pac.USBCTRL_REGS
+        .usb_muxing
+        .write(|w| w.to_phy().set_bit().softcon().set_bit());
+
+    // Get rid of DP pullup override
+    pac.USBCTRL_REGS
+        .usbphy_direct_override
+        .write(|w| w.dp_pullup_en_override_en().set_bit());
```

Comment:
Shouldn't this be clear_bit? Also using modify() instead of write(), since the SDK code reference used hw_clear_alias

Reply:
Ah, that's what I missed! While writing is I found odd that this bit was set both in start and finish.